compiler/luci/pass/src/FuseInstanceNormPass.cpp

```diff
@@ -175,7 +190,53 @@ bool is_instance_mean(luci::CircleMean *mean)
   return mean->keep_dims();
 }
 
-} // namespace
+bool is_instance_mean_v1(luci::CircleMean *mean)
+{
+  //
+  // CHECK 1) input is rank 5
+  //
+  auto input = mean->input();
+  if (not loco::shape_known(input))
+    return false;
+  auto input_shape = loco::shape_get(input).as<loco::TensorShape>();
+  if (input_shape.rank() != 5)
+    return false;
+
+  //
+  // CHECK 2) 'reduction indices' is CircleConst of value [1,2,4], that is HW of NHWC
```

Comment:
I did not get it. The input is 5-D and there should be three reduction incides -- 1, 2 and 4. How does it relate to3 HW and NHWC?

Reply:
If I got it right, 5-d should be always `N x H x W x C x 1`, so 4-th index just removes redundant dimension